### PR TITLE
#169385510 Change spinner appearance

### PR DESCRIPTION
**Pull request by victorkarangwa**
#### What does this PR do? 
- Change the appearance of the spinner
#### Description of Task to be completed?
- This PR changes the appearance of the loading spinner over the app content on API fetch calls
#### How should this be manually tested?
- git clone **`ch-change-spinner-appearance-169385510`** branch
- **`npm install`** dependencies
- **`npm start`** the application
- Search for any specific city or country
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#169385510](https://www.pivotaltracker.com/story/show/169385510)
#### Screenshots (if appropriate)
Screen Shot 2019-10-27 at 21 14 37
Screen Shot 2019-10-27 at 21 15 34

#### Questions:
N/A

### CODE REVIEW

File: src/App.css
Comment (hezzie) [inline src/App.css]: I have a feeling that you should use  % instead px  so that it would be uniform with every screens without using media queries, but good work.
Screen Shot 2019-10-28 at 07 47 09
Screen Shot 2019-10-28 at 07 47 50

Comment (victorkarangwa) [reply src/App.css]: Thanks @hezzie 
This issue has been solved, you can check it out

Comment (hezzie) [reply src/App.css]: Nice work @victorkarangwa

**Review by hezzie — APPROVED** [review]

**Comment by ivymwende** [comment]
Good Job Victor! Just ensure the conflict is solved before merging.

**Review by ivymwende — DISMISSED** [review]

**Comment by NiyoEric** [comment]
Remove all comment.

File: src/App.js
Comment (WillyBoris) [inline src/App.js]: Remove this console.warn then  after merge it

**Comment by kipropbrian** [comment]
Rebase do not merge.

**Review by WillyBoris — APPROVED** [review]

**Review by NiyoEric — APPROVED** [review]

**Pull request merged**
